.golangci.yml

```diff
@@ -0,0 +1,31 @@
+linters:
```

Comment:
IMO : It will be == [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/.golangci.yml) but with the current checks. See:  [here](https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/check-lint.sh#L47-L66)